zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -0,0 +1,190 @@
+          if (pkg.enabled) {
+            defaultVersions[name] = pkg;
+            pkgs.splice(pkgIdx, 1);
+            break;
+          }
+        }
+
+        // show first available version if package is not enabled
+        if (!defaultVersions[name]) {
+          defaultVersions[name] = pkgs[0];
+          pkgs.splice(0, 1);
+        }
+      }
+      $scope.defaultVersions = defaultVersions;
+    };
+
+    var getAllPackageInfo = function() {
+      heliumService.getAllPackageInfo().
+        success(function(data, status) {
+          $scope.packageInfos = data.body;
+          buildDefaultVersionListToDisplay($scope.packageInfos);
+        }).
+        error(function(data, status) {
+          console.log('Can not load package info %o %o', status, data);
+        });
+    };
+
+    var getVisualizationOrder = function() {
+      heliumService.getVisualizationOrder().
+        success(function(data, status) {
+          $scope.visualizationOrder = data.body;
+        }).
+        error(function(data, status) {
+          console.log('Can not get visualization order %o %o', status, data);
+        });
+    };
+
+    $scope.visualizationOrderListeners = {
+      accept: function(sourceItemHandleScope, destSortableScope) {return true;},
+      itemMoved: function(event) {},
+      orderChanged: function(event) {
+        $scope.visualizationOrderChanged = true;
+      }
+    };
+
+    var init = function() {
+      getAllPackageInfo();
+      getVisualizationOrder();
+      $scope.visualizationOrderChanged = false;
+    };
+
+    init();
+
+    $scope.saveVisualizationOrder = function() {
+      var confirm = BootstrapDialog.confirm({
+        closable: false,
+        closeByBackdrop: false,
+        closeByKeyboard: false,
+        title: '',
+        message: 'Save changes?',
+        callback: function(result) {
+          if (result) {
+            confirm.$modalFooter.find('button').addClass('disabled');
+            confirm.$modalFooter.find('button:contains("OK")')
+              .html('<i class="fa fa-circle-o-notch fa-spin"></i> Enabling');
+            heliumService.setVisualizationOrder($scope.visualizationOrder).
+              success(function(data, status) {
+                init();
+                confirm.close();
+              }).
+              error(function(data, status) {
+                confirm.close();
+                console.log('Failed to save order');
+                BootstrapDialog.show({
+                  title: 'Error on saving order ',
+                  message: data.message
+                });
+              });
+            return false;
+          }
+        }
+      });
+    }
+
+    $scope.enable = function(name, artifact) {
+      var confirm = BootstrapDialog.confirm({
+        closable: false,
+        closeByBackdrop: false,
+        closeByKeyboard: false,
+        title: '',
+        message: 'Enable ' + name + '? <div style="color:gray">' + artifact + '</div>',
```

Comment:
Probably it's nit; how about "Do you want to enable + name + ?" for more kind message? :D